js/src/lib/interfaces/Types.ts

```diff
@@ -377,6 +381,10 @@ export const PIPELINE_DATA = ensureRecordOfPipelines({
 				type: "multi-class-image-classification",
 				name: "Multi Class Image Classification",
 			},
+			{
+				type: "binary-image-classification",
```

Comment:
(just curious) If I'm not mistaken this is a supported task in autotrain right ? (according to https://github.com/huggingface/autonlp-backend/blob/49bd7f159120ce8c11b0af0cf55083469ff21d22/zeus/scripts/create_evaluation_project.py#L20-L33)

Reply:
It's soon to be released :) 

In AutoTrain it's called `image_binary_classification`, but we should align it with the `datasets` / Hub taxonomy IMO: https://github.com/huggingface/autonlp-backend/blob/918474d3e0ee914f7ebc50f6d89e20397eacd95e/database/database/enums.py#L126

Reply:
If it's just a matter of having dashes in YAML and underscores in python it's fine I think ;)

Reply:
Yes, it's mostly about reordering the words :)